src/ui/ResultLayout/ResultLayout.ts

```diff
@@ -37,6 +39,7 @@ export class ResultLayout extends Component {
    * The possible valid and supported layout.
```

Comment:
It shows the type `ValidLayout` though, doesn't it ?

If you think it's better to see it directly there, add it in the comments